rpa-bot/1.0/modules/ROOT/edits/installation.adoc

```diff
@@ -0,0 +1,38 @@
+= Installing RPA Bot
+
+Before installing RPA Bot, ensure that your environment meets the minimum hardware and software requirements. After downloading and installing the bot, run the RPA Bot Configurator application to configure its connection and default settings.
+
+== Download and Install RPA Bot
+
+Follow these steps to download and install RPA Bot.
+
+If it is not possible to install the setup as administrator, you need to set the User Account Control setting to *Never notify. Otherwise secure sessions cannot be established and automations will break.
```

Comment:
```suggestion
If it is not possible to install the setup as an administrator, you need to set the User Account Control setting to *Never notify*. Otherwise secure sessions cannot be established and automations will break.
```